proposed/2021/DotNetAudit.md

```diff
@@ -0,0 +1,296 @@
+# dotnet audit & dotnet audit fix
+
+- [Jon Douglas](https://github.com/JonDouglas/)
+- Start Date (2021-04-01)
+- [#8087](https://github.com/NuGet/Home/issues/8087)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+`dotnet audit` & `dotnet audit fix` helps you find, fix, and monitor known security vulnerabilities, deprecated packages, and outdated versions in your .NET projects & solutions.
```

Comment:
It may be crazy but finding unused packages would be awesome.

Reply:
Here's some a community project that does this: https://github.com/spectresystems/snitch

FYI this is a hard problem since a package may be used through reflection. But .NET trimming effort aligns well with this idea.

Reply:
Visual Studio does this today. I think it have much more context & tooling to do so.

![image](https://user-images.githubusercontent.com/1288848/152605101-cf9c4ff2-219f-4cd6-a9ba-9142daa51964.png)

Would be cool for sure! Not sure if feasible until a technical pass.